package.json

```diff
@@ -19,13 +20,13 @@
     "classname"
   ],
   "scripts": {
-    "test": "jest --coverage --no-cache",
+    "test": "jest --coverage --no-cache && tsc --project tests/ts",
     "build": "npm run bundle && npm run minify",
     "bundle": "rollup -i src/index.js -o dist/classwrap.js -m -f umd -n classwrap",
     "minify": "uglifyjs dist/classwrap.js -o dist/classwrap.js --mangle --compress warnings=false --pure-funcs=Object.defineProperty -p relative --source-map dist/classwrap.js.map",
     "bench": "node bench",
     "prepublish": "npm run build",
-    "format": "prettier --semi false --write 'src/**/*.js'",
+    "format": "prettier --semi false --write 'src/**/*.js' 'src/classwrap.d.ts'",
```

Comment:
@Swizz I could be wrong, but I think this would work too:

```
src/**/*.{js,ts}
```

Can you try?

Reply:
It works. Want me to change ?

Reply:
@Swizz Yes, please.